src/main/java/com/points/calculate/CalculatorTree.java

```diff
@@ -0,0 +1,100 @@
+package com.points.calculate;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.commons.lang3.StringUtils;
+
+import com.points.node.twochild.Node;
+
+
+public class CalculatorTree {
+	
+	private Node topNode;
+	
+	public CalculatorTree() {
+		topNode = null;
+	}
+	
+	public CalculatorTree(String equation) {
+		if (StringUtils.isEmpty(equation) ) {
+			topNode = null;
+		} else {
+			topNode = createCalculatorTreeFromString(equation);
+		}		
+	}
+	
+	private static Node createCalculatorTreeFromString(String equation) {	
+		String[] elements = equation.split(" ");
+		List<Node> nodeList = new ArrayList<>();
+		
+		for (String element : elements ) {
+			Node node;
+			try {
+				int num = Integer.parseInt(element);
+				node = new NumberNode(num);
+			} catch (NumberFormatException ex) {
+				try {
+					node = new OperatorNode(Operator.fromString(element));
```

Comment:
If you cannot build the entire calculation; you should probably quit. Giving a result while silently removing an unrecognized value could be very confusing.